eventually/tests/unittests/curriculum/testmodel.py

```diff
@@ -22,11 +22,11 @@ def setUp(self):
         with mock.patch('django.utils.timezone.now') as mock_time:
             mock_time.return_value = TEST_DATE
             custom_user = CustomUser.objects.create(id=1,
-                                                         email='email1@mail.com',
-                                                         password='1234',
-                                                         first_name='1fname',
-                                                         middle_name='1mname',
-                                                         last_name='1lname')
+                                                    email='email1@mail.com',
```

Comment:
Could you change code formatting to:
custom_user=CustomUser.objects.create(
    id=1,
    email='email1@mail.com',
    password='1234',

Reply:
Done. Now is good?